backend/services/implementations/__tests__/classService.test.ts

```diff
@@ -63,10 +65,10 @@ describe("mongo classService", (): void => {
 
   it("create class for valid teachers", async () => {
     // execute
-    const createdClass = await classService.createClass(testClass[0]);
+    const createdClass = await classService.createClass(mockTestClass);
 
     // assert
-    assertResponseMatchesExpected(testClass[0], createdClass);
+    assertResponseMatchesExpected(mockTestClass, createdClass);
```

Comment:
can you update `assertResponseMatchesExpected` to check for the `isActive` field?